eng/common-tests/spelling/Invoke-Cspell.Tests.ps1

```diff
@@ -0,0 +1,41 @@
+Describe 'Tool Version' {
+    BeforeAll {
```

Comment:
@weshaggard -- This is done in lieu of adding a .vscode/cspell.json file to this repo's root.